schemas/actions/function/option/types/number.json

```diff
@@ -13,13 +13,20 @@
     "type": {
       "const": "Number"
     },
-    "min": {
```

Comment:
This wasn't a validation. This was meant for the possible HTML number slider value.
But I guess it's not a problem if we remove this and change it to validations instead.